vagga.yaml

```diff
@@ -22,22 +17,46 @@ containers:
       - !PipConfig
         dependencies: true
         python-exe: python3.5
+      - !NpmInstall [bower]
+
+  gdg-devel:
+    setup:
+      - !Container base
+      - !BuildDeps
+        - git
+        - mercurial
+      - !PipConfig
+        dependencies: true
+        python-exe: python3.5
       - !Depends requirements/common.txt  # As to https://github.com/tailhook/vagga/issues/236
       - !Py3Requirements requirements/dev.txt
-      - !NpmInstall [bower]
-      - !EnsureDir /var/tmp/run
     environ:
       APP_ROOT: /work  # Make sure this is not overriden by .exports
-    volumes:
-      /var/tmp/run: !Tmpfs
-        size: 5Mi
-        mode: 0o1777
+
+  gdg-test:
+    setup:
+      - !Container base
+      - !BuildDeps
+        - git
+        - mercurial
+      - !Depends requirements/common.txt
+      - !Depends requirements/dev.txt
+      - !Py3Requirements requirements/test.txt
+      - !Py3Requirements requirements/test-env.txt
+    environ:
+      BLUEBERRYPY_CONFIG: "{}"
+      NOSE_TESTCONFIG_AUTOLOAD_YAML: "config/test/app.yml"
 
 commands:
   run: !Supervise
-  description: Run app - testing status
-  children:
-    env: !Command
-    container: gdg_dev
-    run: |
-      make run-dev
```

Comment:
Bad idea. You'd have to support separate documentation/process explaining how to run everything on Mac environments.